dotnet/src/Connectors/Connectors.OpenAI/TextToAudio/AzureOpenAITextToAudioService.cs

```diff
@@ -0,0 +1,157 @@
+/// </summary>
+[Experimental("SKEXP0005")]
+public sealed class AzureOpenAITextToAudioService : ITextToAudioService
+{
+    private const string DefaultApiVersion = "2024-02-15-preview";
+
+    private readonly Dictionary<string, object?> _attributes = new();
+    private readonly ILogger _logger;
+    private readonly HttpClient _httpClient;
+
+    private readonly string _deploymentName;
+    private readonly string _endpoint;
+    private readonly string _apiKey;
+    private readonly string? _modelId;
+
+    /// <inheritdoc/>
+    public IReadOnlyDictionary<string, object?> Attributes => this._attributes;
+
+    /// <summary>
+    /// Gets the key used to store the deployment name in the <see cref="IAIService.Attributes"/> dictionary.
+    /// </summary>
+    public static string DeploymentNameKey => "DeploymentName";
+
+    /// <summary>
+    /// Creates an instance of the <see cref="AzureOpenAITextToAudioService"/> connector with API key auth.
+    /// </summary>
+    /// <param name="deploymentName">Azure OpenAI deployment name, see https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource</param>
+    /// <param name="endpoint">Azure OpenAI deployment URL, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="apiKey">Azure OpenAI API key, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="modelId">Azure OpenAI model id, see https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource</param>
+    /// <param name="httpClient">Custom <see cref="HttpClient"/> for HTTP requests.</param>
+    /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to use for logging. If null, no logging will be performed.</param>
+    public AzureOpenAITextToAudioService(
+        string deploymentName,
+        string endpoint,
+        string apiKey,
+        string? modelId = null,
+        HttpClient? httpClient = null,
+        ILoggerFactory? loggerFactory = null)
+    {
+        Verify.NotNullOrWhiteSpace(deploymentName);
+        Verify.NotNullOrWhiteSpace(endpoint);
+        Verify.StartsWith(endpoint, "https://", "The Azure OpenAI endpoint must start with 'https://'");
+        Verify.NotNullOrWhiteSpace(apiKey);
+
+        this._deploymentName = deploymentName;
+        this._endpoint = endpoint;
+        this._apiKey = apiKey;
+        this._modelId = modelId;
+
+        this._httpClient = HttpClientProvider.GetHttpClient(httpClient);
+        this._logger = loggerFactory?.CreateLogger(this.GetType()) ?? NullLogger.Instance;
+
+        this.AddAttribute(DeploymentNameKey, deploymentName);
+        this.AddAttribute(AIServiceExtensions.ModelIdKey, modelId);
+    }
+
+    /// <inheritdoc/>
+    public Task<AudioContent> GetAudioContentAsync(
+        string text,
+        PromptExecutionSettings? executionSettings = null,
+        Kernel? kernel = null,
```

Comment:
As per our agreement, we pass kernel everywhere so all plugins and connectors will be able to work with it. In all our abstractions (chat completion, text generation etc) we accept nullable kernel instance, so specific connector can use it when required.

For this connector, kernel usage is not required, but maybe for `OtherTextToAudioService` in the future it will be required.